sources/osg/Matrix.js

```diff
@@ -652,9 +653,9 @@ var Matrix = {
     postMultRotate: ( function () {
         var r = matrixCreate();
         return function ( matrix, quat ) {
-            Matrix.makeRotateFromQuat( r, quat );
+            Matrix.makeRotateFromQuat( quat, r );
             Matrix.postMult( matrix, r );
-            return matrix;
+            return r;
```

Comment:
humm..Sorry I'm probably a bit slow today,  I don't get it:
The result of doing premult( a, b ) should be equal to postmult( b, a ). Isn't it? 
So if I do what you say,( postmult( r, matrix ) ) then is the same as premult(matrix, r ), which is not the intended way. 

What I'm missing?